fdbserver/include/fdbserver/DataDistribution.actor.h

```diff
@@ -352,7 +352,7 @@ FDB_DECLARE_BOOLEAN_PARAM(MoveKeyRangeOutPhysicalShard);
 
 class PhysicalShardCollection : public ReferenceCounted<PhysicalShardCollection> {
 public:
-	PhysicalShardCollection() : lastTransitionStartTime(now()), requireTransition(false) {}
```

Comment:
Is this change related to the current PR?

Reply:
I removed this change, it was an unrelated change to get rid of compiler warnings.